devolutions-session/src/dvc/status.rs

```diff
@@ -0,0 +1,25 @@
+#[derive(Debug)]
```

Comment:
FYI: I am planning to revisit now proto in follow-up PR, those codes are temporary